README.md

```diff
@@ -77,10 +77,11 @@ Freeside welcome contributions to these lists from staff and students at the Uni
 7. [#HullCSS Discord](https://hullcss.org/discord).
 8. [Hull Gaming Society Discord](https://discord.gg/Rm5gepFQWA).
 9. [Hull Robotics Society Discord](https://discord.gg/pYP5kqE).
-10. [Department of Computer Science and Technology Twitter](https://twitter.com/HullCompSci).
-11. [University of Hull Hub](https://discord.gg/WsEy47d9j8)
```

Comment:
Why remove this?

Reply:
I swapped them around to keep all discord links together, both are still present within the list

![image](https://user-images.githubusercontent.com/32241933/163191325-40b6a027-223d-47a3-a450-43dbd03f6db0.png)